addons/purchase_stock/models/stock_rule.py

```diff
@@ -148,20 +148,20 @@ def _merge_procurements(self, procurements_to_merge):
         for procurements in procurements_to_merge:
             quantity = 0
             move_dest_ids = self.env['stock.move']
-            order_point_id = self.env['stock.warehouse.orderpoint']
+            orderpoint_id = self.env['stock.warehouse.orderpoint']
             for procurement in procurements:
                 if procurement.values.get('move_dest_ids'):
                     move_dest_ids |= procurement.values['move_dest_ids']
-                if not order_point_id and procurement.values.get('order_point_id'):
-                    order_point_id = procurement.values['order_point_id']
+                if not orderpoint_id and procurement.values.get('orderpoint_id'):
+                    orderpoint_id = procurement.values['orderpoint_id']
                 quantity += procurement.product_qty
             # The merged procurement can be build from an arbitrary procurement
             # since they were mark as similar before. Only the quantity and
             # some keys in values are updated.
             values = dict(procurement.values)
             values.update({
                 'move_dest_ids': move_dest_ids,
-                'order_point_id': order_point_id
+                'orderpoint_id': orderpoint_id,
```

Comment:
This commit is correct. However while browsing the code, it seems that orderpoint_id are not passed through the stock.move. Could you check if the orderpoint_id is set with a receipt method in 2 or 3 steps.

Reply:
Okey